pkg/rbac/cache.go

```diff
@@ -22,9 +23,18 @@ type Cache struct {
 
 	// Clients to external APIs.
 	// Defining these here allow the tests to replace with a mock client.
-	authnClient authnv1.AuthenticationV1Interface
-	pool        pgxpoolmock.PgxPool // Database client
-	restConfig  *rest.Config
+	authnClient       authnv1.AuthenticationV1Interface
+	pool              pgxpoolmock.PgxPool // Database client
+	restConfig        *rest.Config
+	dbConnInitialized bool
+}
+
+func (c *Cache) GetDbConnInitialized() bool {
+	return c.dbConnInitialized
+}
+
+func (c *Cache) SetDbConnInitialized(initialized bool) {
```

Comment:
Set should be private, it can only be changed from inside the package.
Get is public.
```suggestion
func (c *Cache) setDbConnInitialized(initialized bool) {
```